.gitignore

```diff
@@ -59,7 +59,6 @@ sysinfo.txt
 nekoyume/compose/results
 
 # Pseudo commandline options.
-/nekoyume/Assets/StreamingAssets/clo.json*
```

Comment:
My bad. I'll restore it.

Reply:
Added. https://github.com/planetarium/NineChronicles/pull/2351/commits/d1b478043bc956435c2e31b7176b8380332b312c